eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs

```diff
@@ -451,7 +452,7 @@ instance (Era era, Script era ~ AlonzoScript era) => EncCBOR (AlonzoTxWitsRaw er
   encCBOR (AlonzoTxWitsRaw vkeys boots scripts dats rdmrs) =
     encode $
       Keyed
-        (\a b c d e f g -> AlonzoTxWitsRaw a b (c <> d <> e) f g)
+        (\a b c d e f g h -> AlonzoTxWitsRaw a b (c <> d <> e <> f) g h)
```

Comment:
I have no idea what the variables here mean

Reply:
When I myself use single letter variable names, my goal is to communicate "you do not need to understand what these things _are_, only that we are doing some kind of very local reshuffling". And by having a very clear ordering (in this case, the alphabet), I'm hoping to make the reshuffling visually easy to see. In other words, I was trying to communicate that "you do not need to know what the variables here mean".

Maybe no one else thinks this way, and full variable names would be less confusing to the majority of people.

The intended interpretation was supposed to be:

"smash the third, fourth, fifth, and sixth elements together, but otherwise keep the ordering the same".

Reply:
I'll close this, but happy to change my style if the majority doesn't like it :)